examples/context-not-managed.cpp

```diff
@@ -0,0 +1,78 @@
+#include "camp/resources.hpp"
```

Comment:
I think this "not managed" could go into camp.

Reply:
I agree, just wasn't sure where to put it in camp yet